src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -304,4 +306,24 @@ public void nerfFishing(PlayerFishEvent event) {
       }
     }
   }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
+
+      for (ItemStack item : quitter.getInventory().getContents()) { //Drop inventory
+        if (item != null) {
+          quitter.getWorld().dropItemNaturally(player.getLocation(), item);
+        }
+      }
+
+      ItemStack[] playerArmor = quitter.getInventory().getArmorContents(); //Drop Armor
```

Comment:
```java
for (ItemStack armor : quitter.getInventory().getArmorContents()) {
  if(armor == null || armor == Material.AIR) continue;
  quitter.getWorld().dropItemNaturally(player.getLocation(), armor);
}

Reply:
Im assuming you meant `armor.getType() == Material.AIR`